Application/Interfaces/IAuthorizationServices.cs

```diff
@@ -12,4 +12,6 @@ public interface IAuthorizationServices
     //Task<bool> IsFacilityManageByCurrentUser(Guid hostelId, Guid userId),
     Task<bool> IsCurrentUserRentTheRoom(Guid roomId, Guid userId);
     Task<RoomEntity> RoomThatManageByCurrentUser(Guid roomId, Guid userId);
+    Task RoomsInAHostelThatManageByCurrentUser(IEnumerable<Guid> roomIds, Guid hostelId, Guid userId);
```

Comment:
Tên hàm sai

Reply:
Managed??

Reply:
RoomThatManageByCurrentUser hàm này là Update Room hả? hay Delete Room

Reply:
validate list room mangage by curren user and in the same hostel

Reply:
Rồi đặt tên vậy sao biết được hàm đó làm gì